github/repos.go

```diff
@@ -738,3 +799,153 @@ func (s *RepositoriesService) License(ctx context.Context, owner, repo string) (
 
 	return r, resp, nil
 }
+
+// GetPullRequestReviewEnforcement gets pull request review enforcement of a protected branch.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/branches/#get-pull-request-review-enforcement-of-protected-branch
+func (s *RepositoriesService) GetPullRequestReviewEnforcement(ctx context.Context, owner, repo, branch string) (*PullRequestReviewsEnforcement, *Response, error) {
+	u := fmt.Sprintf("/repos/%v/%v/branches/%v/protection/required_pull_request_reviews", owner, repo, branch)
```

Comment:
An important issue I've spotted is that all 7 of these URLs begin with a `/`, they should not (it breaks support for GitHub Enterprise).

```Go
u := fmt.Sprintf("repos/...`
```